hedera-node/src/main/java/com/hedera/services/txns/contract/process/EvmTxProcessor.java

```diff
@@ -73,21 +73,21 @@
 
```

Comment:
Since we are including the "refactoring" in this PR, lets remove the `TODO` comments above the `EvmTxProcessor` class definition.